spec/models/issue_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'rails_helper'
+
+describe Issue, type: :model do
+  subject do
+    Issue.new(name: Faker::Name.name_with_middle,
```

Comment:
sample of FactoryGirl+Faker

FactoryGirl.define do
  factory :post do
    title       { Faker::Book.title }
    description { Faker::Lorem.sentence }
    content     { Faker::Lorem.paragraph }
    url         nil
    association :author, factory: :user

    photo { Rack::Test::UploadedFile.new(Rails.root.join('spec', 'files', 'test.png'), 'image/jpg') }

    trait :with_url do
      url { Faker::Internet.url }
    end

    trait :with_link do
      content 'Some content <a href="https://www.google.com">some link</a>'
    end

    trait :with_comments do
      after(:create) do |post, _|
        10.times do
          FactoryGirl.create(:comment, commentable: post, author: post.author)
        end
      end
    end
  end
end